templates/pages/listing.html

```diff
@@ -1,218 +1,63 @@
 {% extends 'layout/base.html' %}
 {% block content %}
-<h3>Listing hope this works</h3>
+<h3>Listing</h3>
 
 <div class="card-deck">
+
+    {% for entry in entries %}
     <div class="card">
-        <a href="{{url_for('entry')}}"><img src=" http://placeimg.com/640/480/any" class="card-img-top"
+        {% if entry.image %}
+        <a href="{{url_for('entry', entry_id = entry._id)}}"><img src="{{ entry.image }}" class="card-img-top"
```

Comment:
Your usage of spaces within templates is a little inconsistent. I would recommend putting them everywhere. :)